R/metrics-quantile.R

```diff
@@ -454,6 +454,45 @@ bias_quantile_single_vector <- function(observed, predicted, quantile, na.rm) {
 }
 
 
+#' @title Absolute Error of the Median (Quantile-based Version)
```

Comment:
This looks like a new function not a test?

Reply:
or you've just copied it for some reason?

Reply:
I just moved it up! That was because the code was under the one-to-one heading, but should be under the many-to-one heading in the file